kernel/configs/arm64_kata_kvm_4.14.x

```diff
@@ -9,7 +9,7 @@ CONFIG_MMU=y
 CONFIG_ARM64_PAGE_SHIFT=12
 CONFIG_ARM64_CONT_SHIFT=4
 CONFIG_ARCH_MMAP_RND_BITS_MIN=18
-CONFIG_ARCH_MMAP_RND_BITS_MAX=24
+CONFIG_ARCH_MMAP_RND_BITS_MAX=33
```

Comment:
/cc @grahamwhaley - x86_64 specifies 32 here whilst ppc64 specifies 33. Should we do 34? :laughing:

Reply:
Heh heh - this is probably quite arch specific, to do with MMU mmaps - so, expect them to maybe be different.
For ref, the x86 kernel config arch files have some defaults if not set (different for 32 and 64 bit - I'll just list the 64bit ones):
- MAX 32
- MIN 28